README.md

```diff
@@ -27,6 +27,22 @@ This product distribution is made available under the [Apache 2.0 license](http:
 
 Portions of this product distribution require certain copyright or other notices to be included with the product distribution. These notices may also appear in individual source files. 
 
+Project Requirements
+---
+- Cocoapods
+- Git
+
+
+Setting Up The Project
+---
+1. Clone the project
+2. Run `sh setup-project.sh` on your terminal.
```

Comment:
The script's jobs seem to be to call `pod install` - which isn't necessary as we commit the pod projects to this repo - and to update the AlfrescoSDK submodule.

I feel this could all be simplified to `git clone --recursive git@github.com:Alfresco/alfresco-ios-app.git` and remove the need for the script completely. I do think that's missing from the README so will make that modification.

Thanks for your contribution.